LinkifiedText/index.jsx

```diff
@@ -49,6 +49,7 @@ LinkifiedText.propTypes = {
       indices: PropTypes.arrayOf(React.PropTypes.number),
     }),
   ),
+  size: PropTypes.string,
```

Comment:
Heya @stevemdixon 😄 This is looking good! I agree with your hunch here of adding the propType with the specific text sizes that are supported. The check is made in `Text` as well but I think it might feel good to make sure `LinkifiedText` props are clear about what configs are supported. 😄